docs/index.adoc

```diff
@@ -28,31 +35,29 @@ image:logo/horizontal.png[picocli the Mighty Tiny Command Line Interface,width=8
 The user manual for the latest release is at http://picocli.info.
 For the busy and impatient: there is also a link:quick-guide.html[Quick Guide].
 
-== Introduction
+== Introducing Picocli
+Picocli is a one-file framework for creating Java command line applications with almost zero code.
 Picocli aims to be the easiest way to create rich command line applications that can run on and off the JVM.
 
-Picocli is a one-file framework for creating Java command line applications with almost zero code.
-Supports a variety of command line syntax styles including POSIX, GNU, MS-DOS and more.
-Generates highly customizable usage help messages with <<ANSI Colors and Styles,ANSI colors and styles>>.
+=== Features
+Picocli supports a variety of command line syntax styles including POSIX, GNU, MS-DOS and more and generates highly customizable usage help messages with <<ANSI Colors and Styles,ANSI colors and styles>>.
 Picocli-based applications can have link:autocomplete.html[command line TAB completion] showing available options, option parameters and subcommands, for any level of nested subcommands.
 Picocli-based applications can be ahead-of-time compiled to a image:https://www.graalvm.org/resources/img/logo-colored.svg[GraalVM]
 <<GraalVM Native Image,native image>>, with extremely fast startup time and lower memory requirements, which can be distributed as a single executable file.
 
-
-image:checksum-usage-help.png[Screenshot of usage help with Ansi codes enabled]
-
 Picocli <<Generate Man Page Documentation,generates beautiful documentation>> for your application (HTML, PDF and Unix man pages).
 
 Another distinguishing feature of picocli is how it aims
 to let users run picocli-based applications without requiring picocli as an external dependency:
 all the source code lives in a single file, to encourage application authors to include it _in source form_.
 
-How it works: annotate your class and picocli initializes it from the command line arguments,
-converting the input to strongly typed values in the fields of your class.
+=== Sample application
+Let's get started by looking at minimal picocli-based command line application. This fully working sample app may be used to calculate the checksum(s) of one or more file(s) given as command line parameter(s):
```

Comment:
Overall, I am not sure about the changes to the Introduction section.
On the one hand, the additional structure of the Features and Sample Application subsections is nice in that it signals what is coming to readers.

On the other hand, naming the subsection "Features" suggests that what is mentioned in that paragraph are the only features that are noteworthy. Or at least the most important ones. I would like the reader's first impression to be different: "this is just a small taste of what is available".

Many users mention that what they like about picocli is how full-featured it is.
I want to enforce and and emphasize that in the user manual. That is also why I used Features instead of Table of Contents as the header for the TOC. Strictly speaking, you are correct: this _is_ the Table of Contents. But by labeling it "Features", followed by 200+ entries, the first impression that people get, looking at this document is "wow, this thing has a _lot_ of features". That is what I was aiming for, at least... :-)

Reply:
> On the one hand, the additional structure of the Features and Sample Application subsections is nice in that it signals what is coming to readers.

That was my intention, and I still consider this an improvement.

> On the other hand, naming the subsection "Features" suggests that what is mentioned in that paragraph are the only features that are noteworthy. Or at least the most important ones. I would like the reader's first impression to be different: "this is just a small taste of what is available".

I see the point. What about "Selected features", "Features (amongst many others)", "Features that make picocli unique, ...

> Many users mention that what they like about picocli is how full-featured it is.
> I want to emphasize that in the user manual. That is also why I used Features instead of Table of Contents as the header for the TOC. Strictly speaking, you are correct: this _is_ the Table of Contents. But by labeling it "Features", followed by 200+ entries, the first impression that people get, looking at this document is "wow, this thing has a _lot_ of features". That is what I was aiming for, at least... :-)

Accepted. We can revert 'Table of contents' back to 'Features', no problem with that.

Reply:
How about **Introduction** for the section, and **Overview** for the first subsection (instead of Features)?